src/main/scala/codacy/docker/api/package.scala

```diff
@@ -11,4 +11,9 @@ package object api extends JsonApi{
     def Value(raw:String):Parameter.Value = Value(Try(Json.parse(raw)).getOrElse(JsString(raw)))
   }
 
-}
\ No newline at end of file
+  implicit class ConfigurationExtensions(config:Configuration.type){
+    def Value(jsValue:JsValue):Configuration.Value = ConfigurationValue(jsValue)
```

Comment:
Intellij 2017.1 applies contextual format depending on the file previous formatting.